test/fixtures_registration.js

```diff
@@ -167,7 +167,7 @@ module.exports = function() {
                             "registered_at": "2015-07-22 00:00:00",
                             "registered_by": "cb245673-aa41-4302-ac47-00000000001",
                             "chew_phone_used": true,
-                            "msg_receiver": "mother",
+                            "receiver_role": "mother",
```

Comment:
`msg_receiver` is the field this value is stored under in the registration but in the identity store it's stored under `receiver_role`